platform/android/MapboxGLAndroidSDKTestApp/src/main/AndroidManifest.xml

```diff
@@ -692,94 +693,105 @@
         </activity>
         <activity
             android:name=".activity.style.HeatmapLayerActivity"
             android:description="@string/description_heatmaplayer"
             android:label="@string/activity_heatmaplayer">
             <meta-data
                 android:name="@string/category"
-                android:value="@string/category_style"/>
+                android:value="@string/category_style" />
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value=".activity.FeatureOverviewActivity" />
+        </activity>
+        <activity
+            android:name=".activity.camera.GestureDetectorActivity"
+            android:label="@string/activity_gesture_detector"
+            android:description="@string/description_gesture_detector">
+            <meta-data
+                android:name="@string/category"
+                android:value="@string/category_camera" />
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
-                android:value=".activity.FeatureOverviewActivity"/>
+                android:value="com.mapbox.mapboxsdk.testapp.activity.FeatureOverviewActivity" />
         </activity>
 
         <!-- For Instrumentation tests -->
         <activity
             android:name=".activity.style.RuntimeStyleTestActivity"
-            android:screenOrientation="portrait"/>
+            android:screenOrientation="portrait" />
         <activity
             android:name=".activity.style.RuntimeStyleTimingTestActivity"
-            android:screenOrientation="portrait"/>
+            android:screenOrientation="portrait" />
         <activity
             android:name=".activity.espresso.EspressoTestActivity"
-            android:screenOrientation="portrait"/>
+            android:screenOrientation="portrait" />
         <activity
             android:name=".activity.style.FillExtrusionStyleTestActivity"
-            android:screenOrientation="portrait"/>
+            android:screenOrientation="portrait" />
         <!-- Configuration Settings -->
         <meta-data
             android:name="com.mapbox.TestEventsServer"
-            android:value="api-events-staging.tilestream.net"/>
+            android:value="api-events-staging.tilestream.net" />
         <meta-data
             android:name="com.mapbox.TestEventsAccessToken"
-            android:value="pk.eyJ1IjoiYmxzdGFnaW5nIiwiYSI6ImNpdDF3OHpoaTAwMDcyeXA5Y3Z0Nmk2dzEifQ.0IfB7v5Qbm2MGVYt8Kb8fg"/>
+            android:value="pk.eyJ1IjoiYmxzdGFnaW5nIiwiYSI6ImNpdDF3OHpoaTAwMDcyeXA5Y3Z0Nmk2dzEifQ.0IfB7v5Qbm2MGVYt8Kb8fg" />
 
         <!-- Comment out this setting to switch to external storage (and disable internal) in your app -->
         <!-- <meta-data -->
         <!-- android:name="com.mapbox.SetStorageExternal" -->
         <!-- android:value="true" /> -->
     </application>
 
-</manifest>
+</manifest>
```

Comment:
I think we need to sync our checkstyle configuration for the formatter. To remove the unnecessary changes introduced in this file.

Reply:
I thought it's a good time as any other to actually fix the formatting of the Manifest file. Let me know if you'd still rather like to adjust the checkstyle configuration.